src/util/util.c

```diff
@@ -41,7 +41,7 @@ int metres_to_feet(int metres)
 
 int mph_to_kmph(int mph)
 {
-	// 1 mph = 1.60934 kmph
+	// 1 mph = 1.60934 kph
```

Comment:
I guess both are correct, however I went with the more common one.
I suppose this is up to personal opinion, though it's just a comment line, so it doesn't really matter anyways I suppose.
From now on I'll accept both forms when spellchecking.
Thank you.